votingpool/db.go

```diff
@@ -56,13 +59,79 @@ type dbSeriesRow struct {
 	privKeysEncrypted [][]byte
 }
 
+// getUsedAddrBucketID returns the used addresses bucket ID for the given series
+// and branch. It has the form seriesID:branch.
+func getUsedAddrBucketID(seriesID uint32, branch Branch) []byte {
+	return bytes.Join(
```

Comment:
nitpick: The rest of function code base uses as much of the line as possible before wrapping to the next.  There are a few more of these, but I won't comment on every one.